crypto/fipsmodule/bn/montgomery.c

```diff
@@ -418,6 +461,78 @@ static int bn_mod_mul_montgomery_fallback(BIGNUM *r, const BIGNUM *a,
   return ret;
 }
 
+
+#if defined(OPENSSL_BN_ASM_MONT)
+
+static void montgomery_s2n_bignum_mul_mont(BN_ULONG *rp, const BN_ULONG *ap,
```

Comment:
Can we explicitly state in a comment which parameter sizes are supported by this function?

Reply:
In general, all parameter sizes whose number of words is a multiple of 8 and fits in buffer. For specific sizes where num = 32 or 64 (if 64-bit, 1024bit or 2048bit), this function uses faster implementations (bignum_k{sqr,mul}_16_32, bignum_k{sqr,mul}_32_64). Will be commented in the next commit